### PR TITLE
Vultr: Create the Vultr Guide

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

Writing a scenario guide for Vultr cloud platform

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

  - vultr

##### ANSIBLE VERSION

  - devel

##### ADDITIONAL INFORMATION

  - N/A

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_vultr.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+Configuration is read in that order:
+
+  - Environment Variables (eg. `VULTR_API_KEY`, `VULTR_API_TIMEOUT`)
+  - File specified by environment variable `VULTR_API_CONFIG`
+  - `vultr.ini` file located in current working directory
+  - `$HOME/.vultr.ini`
+
+
+Ini file are structured this way::
+
```
Comment (resmo) [inline docs/docsite/rst/scenario_guides/guide_vultr.rst]: I would add code block for better syntax highlighting:

.. code-block:: ini

File: docs/docsite/rst/scenario_guides/guide_vultr.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+If a similar output displays then everything is setup properly, else please ensure the proper `VULTR_API_KEY` has been specified and that Access Control on Vultr > Account > API page are accurate.
+
+
+Usage
+-----
+
+Since `Vultr <https://www.vultr.com>`_ offers a public API, the execution of the module to manage the infrastructure on their platform will happen on localhost. This translates to::
+
```
Comment (resmo) [inline docs/docsite/rst/scenario_guides/guide_vultr.rst]: .. code-block:: yaml

File: docs/docsite/rst/scenario_guides/guide_vultr.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+Dynamic Inventory
+-----------------
+
+Ansible provides a dynamic inventory plugin for `Vultr <https://www.vultr.com>`_.
+The configuration process is exactly the same as the one for the modules.
+
+To be able to use it one needs to enable it first by specifying the following in the `ansible.cfg` file::
+
```
Comment (resmo) [inline docs/docsite/rst/scenario_guides/guide_vultr.rst]: .. code-block:: ini

File: docs/docsite/rst/scenario_guides/guide_vultr.rst [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  ---
+  plugin: vultr
+
+To list the available hosts one can simply run::
+
+  #> ansible-inventory - i vultr.yml
+
+
+This allows one - for example - to take action on nodes grouped by location or OS name::
+
```
Comment (resmo) [inline docs/docsite/rst/scenario_guides/guide_vultr.rst]: .. code-block:: yaml

File: docs/docsite/rst/scenario_guides/guide_vultr.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+Ansible provides a dynamic inventory plugin for `Vultr <https://www.vultr.com>`_.
+The configuration process is exactly the same as the one for the modules.
+
+To be able to use it one needs to enable it first by specifying the following in the `ansible.cfg` file::
+
+  [inventory]
+  enable_plugins=vultr
+
+And provide a configuration file to be used with the plugin, the minimal configuration file looks like this::
+
```
Comment (resmo) [inline docs/docsite/rst/scenario_guides/guide_vultr.rst]: .. code-block:: yaml

**Review by resmo — COMMENTED** [review]
no blockers. only some styles, lgtm

File: docs/docsite/rst/scenario_guides/guide_vultr.rst [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+Configuration
+-------------
+
+Vultr modules offer a rather flexible way with regard to configuration.
+
+Configuration is read in that order:
+
+  - Environment Variables (eg. `VULTR_API_KEY`, `VULTR_API_TIMEOUT`)
```
Comment (gundalow) [inline docs/docsite/rst/scenario_guides/guide_vultr.rst]: Remove leading spaces before `-`

For inline code in RST it's double backticks

```
Environment Variables (eg. ``VULTR_API_KEY``, ``VULTR_API_TIMEOUT``)

```

File: docs/docsite/rst/scenario_guides/guide_vultr.rst [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+          size: 10
+          region: New Jersey
+
+
+From that point on, only you creativity is the limit. Make sure to read the documentation of the `available modules <https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#vultr>`_.
+
+
+Dynamic Inventory
+-----------------
+
```
Comment (gundalow) [inline docs/docsite/rst/scenario_guides/guide_vultr.rst:109]: Link to inventory script?

Comment (resmo) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst:109]: the inventory is currently in an open PR #44400

File: docs/docsite/rst/scenario_guides/guide_vultr.rst (hunk 1 above)
Comment (Spredzy) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst]: done

File: docs/docsite/rst/scenario_guides/guide_vultr.rst (hunk 2 above)
Comment (Spredzy) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst]: done

File: docs/docsite/rst/scenario_guides/guide_vultr.rst (hunk 3 above)
Comment (Spredzy) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst]: done

File: docs/docsite/rst/scenario_guides/guide_vultr.rst (hunk 4 above)
Comment (Spredzy) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst]: done

File: docs/docsite/rst/scenario_guides/guide_vultr.rst (hunk 5 above)
Comment (Spredzy) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst]: done

File: docs/docsite/rst/scenario_guides/guide_vultr.rst (hunk 6 above)
Comment (Spredzy) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst]: done

File: docs/docsite/rst/scenario_guides/guide_vultr.rst (hunk 7 above)
Comment (Spredzy) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst:109]: @gundalow as stated by @resmo its a PR in progress. Hopefully will get merged soon.

By the way to you know if inventory plugings are impacted by core freeze?

Comment (resmo) [reply docs/docsite/rst/scenario_guides/guide_vultr.rst:109]: @Spredzy Because it is not a core plugin, I would say no impact

**Comment by resmo** [comment]
rebuild_merge

**Pull request merged**